pom.xml

```diff
@@ -261,9 +246,6 @@
           <argLine>
             ${surefireArgLine}
           </argLine>
-          <!-- fork a new JVM to isolate test suites, especially important with singletons -->
-          <forkCount>${cpu.count}</forkCount>
-          <reuseForks>false</reuseForks>
```

Comment:
As witnessed by @Kavindu-Dodan the test suite is actually 3x faster without any forking at all :shrug:

Reply:
This observation can be explained with the surefire documentation [1]

> The parameter reuseForks is used to define whether to terminate the spawned process after one test class and to create a new process for the next test in line (reuseForks=false), or whether to reuse the processes to execute the next tests (reuseForks=true).

`forkCount` helped with creating N parallel forks but, given that `reuseForks` was false, tests did not reuse them. This means, while tests ran in parallel forks, new tests require a new process, adding a delay between executions. 

[1] - https://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html#forked-test-execution

Reply:
Thanks!

I guessed it was something like that. It makes sense that starting up a bunch of JVMs would slow things down.

I did just try `<forkCount>1C</forkCount>` (which means 1 x number-of-cores), while allowing fork re-use, and the speed of the suite is unchanged from the default... so it doesn't slow things down but doesn't really help much.